Content.Server/Silicons/Borgs/BorgSystem.MMI.cs

```diff
@@ -40,10 +41,17 @@ private void OnMMIEntityInserted(EntityUid uid, MMIComponent component, EntInser
         linked.LinkedMMI = uid;
         Dirty(uid, component);
 
+        _appearance.SetData(uid, MMIVisuals.BrainPresent, true);
+
         if (_mind.TryGetMind(ent, out var mindId, out var mind))
+        {
             _mind.TransferTo(mindId, uid, true, mind: mind);
 
-        _appearance.SetData(uid, MMIVisuals.BrainPresent, true);
+            var job = EnsureComp<JobComponent>(mindId);
+
+            linked.OldJob = job.Prototype;
+            job.Prototype = "Borg";
+        }
```

Comment:
This should use events / separate component for job changes on mind change instead of hardcoding borgs.